WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+import Yosemite
+
+struct AddEditCoupon: View {
```

Comment:
Nit: maybe we should add a short comment explaining that the view is reused for both adding and editing coupons here?